spec/std/system_spec.cr

```diff
@@ -4,16 +4,21 @@ require "system"
 describe System do
   describe "hostname" do
     it "returns current hostname" do
-      shell_hostname = `hostname`.strip
-      $?.success?.should be_true # The hostname command has to be available
+      shell_hostname = {{ `hostname`.stringify }}.strip # Workaround for inability to execute shell commands on Windows.
```

Comment:
Maybe add a TODO so this can be changed once `Process` is ported?

Reply:
This shouldn't be changed in the first place. The spec works fine as is on Unix and it will on Windows as soon as `Process` is ported.

Reply:
I've changed this back, though I thought on gitter you suggested changing this to a macro.